lib/python/ensembl/brc4/runnable/core_server.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import re
+from typing import List
+
+import mysql.connector
+
+
+class CoreServer:
+    """Basic interface to a MySQL server with core databases.
+
+    Allows to get a list of databases and provides access to them.
+
+    Args:
+    host
+    port
+    user
+    password (optional)
```

Comment:
Apologies, should have done the previous comment in two separate ones:
```suggestion
    Attributes:
        host
        port
        user
        password (optional)
```